media/css/mozorg/contentservices.less

```diff
@@ -0,0 +1,207 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../sandstone/sandstone-resp.less";
+
+/* Hide fields to be shown with JS later */
+
+.type-field,
+.state-field {
+    display: none;
+}
+
+.cloaked {
+    display: none;
+    visibility: hidden;
+}
+
+
+#wrapper {
+    width: 100%;
+}
+
+#masthead,
+#main-content,
+.billboard {
+    width: @widthDesktop - (@gridGutterWidth * 2);
+    margin-left: auto;
+    margin-right: auto;
+}
+
+#main-content {
+    .callout-content;
+    position: relative;
+    margin: 100px auto @baseLine;
+    .open-sans;
+
+    .main {
+        .span(6);
+        float: left;
+        clear: left;
+        margin-bottom: @baseLine;
+
+        .lede {
+            .open-sans-light;
+            .font-size(20px);
+            line-height: 1.3;
+            margin-bottom: @baseLine * 2;
+        }
+
+        .start {
+            text-align: center;
+            h4 {
+                text-shadow: none;
+            }
+
+            .button-flat {
+                min-width: 240px;
+                margin-bottom: 10px;
+                padding: @baseLine;
+                font-size: @largeFontSize;
+            }
+        }
+    }
+}
+
+.video {
+    .span(6);
+    float: right;
+    margin-top: -50px;
+    margin-bottom: @baseLine;
+    #welcome-video video {
```

Comment:
Nit - don't really need to nest an id selector (though it is nice for organizational purposes).